ProcessMaker/processmaker
# Implement Unit Tests for Screen Templates API CRUD Functionality

This PR adds unit tests to validate the API CRUD functionality for screen templates. These tests ensure that the create, update, and delete operations for screen templates are working as expected.

## How to Test

1. Ensure you have the necessary testing environment set up.
2. Run the unit tests associated with screen templates `phpunit tests/Feature/Templates/Api/ScreenTemplateTest.php`
3. Validate that the tests pass without errors or failures.

## Related Tickets & Packages
 - [FOUR-13487](https://processmaker.atlassian.net/browse/FOUR-13487)

ci:next

.

## Code Review Checklist


[FOUR-13487]: https://processmaker.atlassian.net/browse/FOUR-13487?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
